images/runtime/php-fpm/8.1/base.bullseye.Dockerfile

```diff
@@ -0,0 +1,137 @@
+FROM oryxdevmcr.azurecr.io/private/oryx/php-fpm-8.1
```

Comment:
These should be gitignored now, so we can safely remove these `base.*.Dockerfiles` from the git index with `git rm --cached <file>`